src/azure-cli-core/setup.py

```diff
@@ -51,30 +51,28 @@
     'License :: OSI Approved :: MIT License',
 ]
 
-# TODO These dependencies should be updated to reflect only what this package needs
 DEPENDENCIES = [
-    'adal>=1.2.0',
-    'argcomplete>=1.8.0',
+    'adal~=1.2.0',
+    'argcomplete~=1.8.0',
```

Comment:
Mind making these `adal~=1.2` and `argcomplete~=1.8`?

Reply:
They were in the original PR, oopsies